lavamoat/webpack/policy-override.json

```diff
@@ -0,0 +1,328 @@
+        "HTMLIFrameElement": true,
+        "DocumentFragment": true,
+        "Function.prototype": true,
+        "Object.prototype": true,
+        "Error.prototype": true
+      }
+    },
+    "scheduler": {
+      "globals": {
+        "MessageChannel": true,
+        "clearTimeout": true,
+        "setTimeout": true,
+        "window": true,
+        "Error": true,
+        "Object": true,
+        "Function.prototype": true,
+        "Object.prototype": true
+      }
+    },
+    "bitcore-lib": {
+      "globals": {
+        "Error": true,
+        "Object": true,
+        "Math": true,
+        "Buffer": true,
+        "Function.prototype": true,
+        "Object.prototype": true,
+        "Error.prototype": true,
+        "Math.prototype": true,
+        "console": true,
+        "Function": true,
+        "Array": true,
+        "Number": true,
+        "BigInt": true
+      },
+      "packages": {
+        "bn.js": true,
+        "elliptic": true,
+        "buffer": true
+      }
+    },
+    "bn.js": {
+      "globals": {
+        "Error": true,
+        "Object": true,
+        "Math": true,
+        "Buffer": true,
+        "Number": true,
+        "Array": true,
+        "Function": true,
+        "Function.prototype": true,
+        "Object.prototype": true,
+        "Array.prototype": true
+      }
+    },
+    "elliptic": {
+      "globals": {
+        "Error": true,
+        "Object": true,
+        "Math": true,
+        "Buffer": true,
+        "Function": true,
+        "Function.prototype": true,
+        "Object.prototype": true,
+        "Error.prototype": true,
+        "Math.prototype": true,
+        "Array": true,
+        "Number": true
+      },
+      "packages": {
+        "bn.js": true
+      }
+    },
+    "axios": {
+      "globals": {
+        "process": true,
+        "console": true,
+        "Error": true,
+        "Object": true,
+        "XMLHttpRequest": true,
+        "FormData": true,
+        "Blob": true,
+        "window": true,
+        "Object.prototype": true,
+        "Function.prototype": true,
+        "Object.defineProperty": true,
+        "Object.getOwnPropertyDescriptor": true,
+        "Symbol": true,
+        "Symbol.toStringTag": true,
+        "__esModule": true,
+        "require": true,
+        "module": true,
+        "exports": true,
+        "Function": true,
+        "Array": true,
+        "String": true,
+        "Promise": true,
+        "Array.prototype": true,
+        "String.prototype": true
+      },
+      "packages": {
+        "follow-redirects": true,
+        "form-data": true,
+        "proxy-from-env": true,
+        "url": true,
+        "http": true,
+        "https": true,
+        "assert": true,
+        "stream": true,
+        "util": true
+      }
+    },
+    "@hathor/wallet-lib": {
+      "globals": {
+        "process": true,
+        "console": true,
+        "Error": true,
+        "Object": true,
+        "XMLHttpRequest": true,
```

Comment:
We send requests only through axios. Does it make sense to remove this and allow only in axios?

Reply:
It does make sense, removed XMLHttpRequest and some others that are not being used